snapcraft/internal/project_loader/_parts_config.py

```diff
@@ -201,9 +201,7 @@ def load_part(self, part_name, plugin_name, part_properties):
         for source in sources:
             repo.Repo.install_source(source)
 
-        # TODO: rename with migration strategy.
-        repo_dir = path.join(self._project.parts_dir, part_name, "ubuntu")
-        stage_packages_repo = repo.Repo(repo_dir)
+        stage_packages_repo = repo.Repo
```

Comment:
repo_dir is used in PluginHandler, so you might need to adapt some tests or actual code.